backend/app/views/spree/admin/promotion_code_batches/download.csv.ruby

```diff
@@ -0,0 +1,6 @@
+CSV.generate do |csv|
+  csv << ['Code']
```

Comment:
Normally this header should be translatable, but this view could also be overwritten in the host app. So, I'm not super keen about it.